SPECS/glib/glib.spec

```diff
@@ -1,52 +1,54 @@
-Summary:	Low-level libraries useful for providing data structure handling for C.
-Name:		glib
-Version:	2.58.0
-Release:        6%{?dist}
-License:	LGPLv2+
-URL:		https://developer.gnome.org/glib/
-Group:		Applications/System
+%define sha1 glib=c00e433c56e0ba3541abc5222aeca4136de10fb8
+Summary:        Low-level libraries useful for providing data structure handling for C.
+Name:           glib
+Version:        2.58.0
+Release:        7%{?dist}
+License:        LGPLv2+
```

Comment:
It looks like we haven't checked the license for this one for some reason. Could you verify that it's really `LGPLv2+` and add the phrase "License verified" to the changelog? This should only require checking the contents of the `COPYING` file.